index.js

```diff
@@ -1,6 +1,13 @@
 'use strict';
 
-var assert = require('assert');
+var assert = {
+  ok(cond, msg) {
```

Comment:
Could probably just be `function assert(cond, msg) {` as we only use the default export of `assert` here.